src/runtime_src/core/tools/xbmgmt2/OO_Config.h

```diff
@@ -29,6 +29,16 @@ class OO_Config : public OptionOptions {
  private:
   std::string m_device;
   bool m_help;
+  bool daemon;
```

Comment:
Please follow the previous author of this code base and pre-append "m_" to the internal variables.